.github/workflows/auto-approve.yml

```
@@ -0,0 +1,13 @@
+name: Auto Approve Dependabot PRs
+
+on:
+  pull_request
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@v2.0.0
+      if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
```

```suggestion
      if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]' || github.actor == 'notlee'
```

@JakeChampion not ideal but let's test out with my github username and assume all is well for dependabot if my pr to remove this is automerged.